doc/sphinx/language/core/modules.rst

```diff
@@ -1063,3 +1063,79 @@ while noting a few exceptional commands for which :attr:`local` and
       attribute because the transitive nature of file loading gives
       the user little control. We recommend using the :attr:`export`
       locality attribute where it is supported.
+
+The following table sums up the locality of vernacular commands in modules, when
+outside the module.
+
+.. list-table::
+  :header-rows: 1
+
+  * - :cmd:`Command`
+    - without attribute, not imported
+    - without attribute, imported
+    - :attr:`local`
+    - :attr:`export`
+    - :attr:`global`
+
+  * - :cmd:`Definition`, :cmd:`Lemma`, :cmd:`Axiom`, ...
+    - available with qualified name
+    - available with short name
+    - available with qualified name
+    - attribute not supported
+    - same as without attribute
+
+  * - :cmd:`Notation`
+    - not available outside
+    - available outside
+    - available with qualified name
```

Comment:
Isn't this Local Notation? Then it should be not available outside

Reply:
Yes, mistake on my part

Reply:
Changed